client/layout/logged-out.jsx

```diff
@@ -339,6 +339,7 @@ export default withCurrentRoute(
 				( ! isJetpackLogin &&
 					Boolean( currentQuery?.client_id ) === false &&
 					Boolean( currentQuery?.oauth2_client_id ) === false &&
+					! oauth2Client &&
```

Comment:
I can see that `isBlazePro` is true on this screen, so maybe that would be more accurate? I wonder whether `oauth2Client` might affect other screens...

```suggestion
					! isBlazePro &&
```

Reply:
Oh, good call! That'll help make it more consistent, too 👍